Orange/widgets/gui.py

```diff
@@ -552,6 +552,14 @@ def widgetBox(widget, box=None, orientation='vertical', margin=None, spacing=4,
     return b
 
 
+def hBox(*args, **kwargs):
+    return widgetBox(orientation="horizontal", *args, **kwargs)
```

Comment:
Sorry, didn't see this before. I don't think this syntax is correct. Positional arguments can't follow keyword arguments.
Should be `return widgetBox(*args, orientation='horizontal', **kwargs)`.